app/js/controllers/gesture.js

```diff
@@ -0,0 +1,79 @@
+/*global Controller*/
+
+export default class GestureController extends Controller {
+  constructor(options) {
+    super(options);
+
+    var firstTouchStartEvt = null;
+    var isForwarding = false;
+
+    window.addEventListener('touchstart', (evt) => {
+      if (evt.touches.length === 1) {
+        firstTouchStartEvt = evt;
+        return;
+      }
+
+      if (evt.touches.length !== 2) {
+        return;
+      }
+
+      var iframe = document.querySelector('#windows > .active iframe');
+      iframe.sendTouchEvent.apply(iframe, unsynthetizeEvent(firstTouchStartEvt));
+      iframe.sendTouchEvent.apply(iframe, unsynthetizeEvent(evt));
+
+      isForwarding = true;
+    });
+
+    window.addEventListener('touchmove', (evt) => {
+      if (!isForwarding) {
+        return;
+      }
+
+      var iframe = document.querySelector('#windows > .active iframe');
```

Comment:
We should store this selector somewhere instead of using two equal static strings.